core/io/resource_format_binary.cpp

```diff
@@ -980,6 +980,8 @@ ResourceInteractiveLoaderBinary::ResourceInteractiveLoaderBinary() :
 		f(NULL),
 		error(OK),
 		stage(0) {
+	ver_format = 0;
+	importmd_ofs = 0;
```

Comment:
Why are some of these before the `{` and some after it?

Reply:
Initializer list(before {) require to have all variables in same order as in class definition, so for me it was easier to write such code(in master initialization was moved from constructor)